### ISSUE TITLE
Additional stylesheets are empty

### ISSUE THREAD

**Issue by MSchmidt** [colinta/teacup#151]
I cannot create more than one stylesheet.

Whenever I create an additional one (doesn't matter whether in same or different file) it just looks like this:

``` ruby
:details_screen=>Teacup::Stylesheet[:details_screen] = {}
```

Empty hash.

**Comment by MSchmidt** [comment]
Hmm ok there is more to it. The issue only exists because I'm adding another subview with it's own styles defined within it's controller. The second View in `DetailViewController` won't get styled at all.

``` ruby
class RootViewController < UIViewController
  stylesheet :root_screen

  def viewDidLoad
    @detail_view_controller = DetailViewController.new
    self.addChildViewController(@detail_view_controller)
    @detail_view_controller.didMoveToParentViewController(self)

    layout(self.view, :root) do
      subview(@detail_view_controller.view)
    end
  end
end
```

``` ruby
class DetailViewController < UIViewController
  stylesheet :details_screen

  def viewDidLoad
    super

    layout(self.view, :root)
  end
end
```

**Comment by colinta** [comment]
Fixed in 2.2.1.

Creating a project to reproduce this error took some time, and for a while I had "false negatives" (code that I thought should reproduce your error, but ended up working correctly).  The sample code helped, but please consider including a repo or spec that reproduces the error in the future so I can solve it quicker.

Thanks for the bug report!
